cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -645,7 +648,42 @@ private TwillController startTwillApplication(TwillRunnerService twillRunner) {
                                                                                 getSystemServiceInstances()));
 
         if (cConf.getBoolean(Constants.COLLECT_CONTAINER_LOGS)) {
-          preparer.addLogHandler(new PrinterLogHandler(new PrintWriter(System.out)));
+          preparer.addLogHandler(new LogHandler() {
+            @Override
+            public void onLog(LogEntry entry) {
+              if (entry.getThrowable() == null) {
+                LOG.info(entry.getMessage());
```

Comment:
This will print the class as `MasterServiceMain` instead of the class from which the log message originated from, right?